LICENSE

```diff
@@ -3,7 +3,7 @@ Business Source License 1.1
 Parameters
 
 Licensor:             Lightbend, Inc.
-Licensed Work:        Akka Persistence JDBC 5.2.x
+Licensed Work:        Akka Persistence JDBC 5.2.1
                       The Licensed Work is (c) 2022 Lightbend Inc.
```

Comment:
also check the license url link in the build (that ends up in the pom.xml)

Reply:
Pulled in the auto-linking for that: [5f1bb44](https://github.com/akka/akka-persistence-jdbc/pull/725/commits/5f1bb44c699151b64fadaa18df257e457a39bbff)